src/devices/DCMotor/samples/README.md

```diff
@@ -1,55 +0,0 @@
-# DC Motor Controller sample
-
-![schematics](dcmotor_bb.png)
```

Comment:
consider moving these to the main README

Reply:
they all have been move. The README.md still in /samples were left overs.